config/config-network.yaml

```diff
@@ -182,3 +183,14 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     internal-encryption: "false"
+
+    # internal-trust indicates whether internal traffic is encrypted and mutual trust is achieved.
+    # Using internal-trust protect against MITM attacks
+    # If this is "true", the following traffic are encrypted:
```

Comment:
I think this is "the following traffic verifies client authentication with mutual TLS", not simply encrypted.